src/ucp/core/ucp_ep.h

```diff
@@ -392,8 +392,12 @@ struct ucp_ep_config {
         ucp_rndv_thresh_t          am_thresh;
         /* Total size of packed rkey, according to high-bw md_map */
         size_t                     rkey_size;
-        /* Remote memory domains which support rkey_ptr */
+        /* Remote memory domains which support rkey_ptr, excluding
+         * rkey_ptr_lane */
```

Comment:
the name rkey_ptr_dst_mds is somewhat confusing; need something in the name to indicate that rkey_ptr_lane is excluded. maybe something like proto_rndv_rkey_skip_md_map?